Gemfile

```diff
@@ -4,3 +4,6 @@ gem 'test-kitchen', '1.20.0'
 gem 'kitchen-docker', '2.6.0'
 gem 'kitchen-ansible', '0.48.1'
 gem 'net-ssh', '4.2.0'
+gem 'kitchen-vagrant', 	'1.3.0'
```

Comment:
Thanks for adding these in, can you also checkin the updated `Gemfile.lock`?

Reply:
Fixed